templates/compositions/ec-component-navigation-list/navigation-list.html.twig

```diff
@@ -37,5 +38,16 @@
       {% endfor %}
     </ul>
   {% endif %}
+  {% if secondary_links is not empty and secondary_links is iterable %}
```

Comment:
Let's remove this, it is eccessive
```
 and secondary_links is iterable
```

Reply:
I removed it as well from the previous statement.